tests/test_login.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import config
+from flask import json
+from tests import api_utils
+from app.pcasts.dao import users_dao
+from tests.test_case import *
+from app import constants # pylint: disable=C0413
+
+class LoginTestCase(TestCase):
+
+  def setUp(self):
+    super(LoginTestCase, self).setUp()
+    Session.query.delete()
+    db_session_commit()
+
+
+  def test_facebook_login(self):
+
```

Comment:
I don't think we've been consistent with new line after the method declaration, but in most of the other tests we dont have this line here